src/doc/man/cargo-package.md

```diff
@@ -98,6 +98,12 @@ or the license).
 Allow working directories with uncommitted VCS changes to be packaged.
 {{/option}}
 
+{{#option "`--exclude-lockfile`" }}
+Don't include the lock file when packaging.
+
+This flag is not for general use. Always consider packaging with a lockfile.
```

Comment:
Not sure if we really want to mention this. People using `cargo package` usually are already in an unusual workflow.

Reply:
I think it might be worth giving a caution on this though we should probably give some semblance of a reason

> This flag is not for general use as some tools may expect a lockfile to be present (e.g. `cargo install --locked`).
> Consider other options before using this.

`cargo-semver-checks` is also something else that would benefit from the lockfile but that feels a bit odd to call out.

Reply:
Adopted your suggestion with just one slight tweak. Thanks!